src/amber/cli/templates/template.cr

```diff
@@ -96,6 +97,11 @@ module Amber::CLI
           raise "Auth not supported for crecto yet"
         end
         Auth.new(name, fields).render(directory, list: true, color: true)
+      when "error"
+        puts "Rendering Error Template"
+        actions = ["forbidden", "not_found", "internal_server_error"]
```

Comment:
The initial implementation, there is no customization allowed - just these 3 actions will be generated.